src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.Cart/OrderDetails/OrderDetailsController.ts

```diff
@@ -40,7 +49,17 @@ module Orckestra.Composer {
                             .fail(() => {
                                 this.Loading = false;
                             });
-                    } 
+                    },
+                    reload(){
+                        window.location.reload()
+                    },
+                    cancelOrderConfirm(event: JQueryEventObject, orderNumber: string) {
+                        this.OrderNumber = orderNumber;
+                        this.Modal.cancelOrderModal.openModal(event);
+                    },
+                    cancelOrder(){
+                        self.orderService.cancelOrder(this.OrderNumber); 
```

Comment:
add before:
 if(this.Loading) return;
                        this.Loading = true;